packages/untp-test-suite/src/core/process-test-suite.ts

```diff
@@ -1,22 +1,22 @@
-import { hasErrors } from './services/index.js';
-import { ConfigContent, TestSuite, TestSuiteResult } from './types/index.js';
-import { readFile, validateCredentialConfigs } from './utils/common.js';
+import path from 'path';
+import { TestSuite } from './types/index';
+import { readConfigFile, readFile } from './utils/common.js';
+import { dynamicLoadingSchemaService } from './services/dynamic-loading-schemas/loadingSchema.service.js';
+import { hasErrors } from './services/json-schema/validator.service.js';
 
-export const processTestSuite: TestSuite = async (credentialConfigsPath) => {
-  const credentialConfigs = await readFile<ConfigContent[]>(credentialConfigsPath);
-  validateCredentialConfigs(credentialConfigs);
-
-  const testSuiteResultPromises = credentialConfigs.map<Promise<TestSuiteResult>>(async (credentialConfig) => {
+export const processTestSuite: TestSuite = async () => {
```

Comment:
I think this function should have a parameter to specify the path of the credential configuration file we want to read. This would make the function more dynamic and easier to reuse. For example, in the `untp test` command, which calls this function, there could be an option for the user to specify the path of the credential configuration file (--config \<path\>) they want to use when running the test suite.